lib/conditionals.md

```diff
@@ -5,8 +5,9 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+ A conditional is what controls the flow of a program. A contional allows one to give a program conditions that, 
+ when met, execute the program one way, or if not met, allow the program to run another way (for example, If, else, elseif, etc...).
```

Comment:
You cited examples of control flow like `if`, `else`, and `elsif`. Two followup questions for you:

1. How specifically are conditional statements used with `else` as opposed to `elsif`? Feel free to use code examples if it helps explain.
2. What is one other example of control flow where a conditional might be used?